CODEOWNERS

```
@@ -0,0 +1 @@
+* @jerivas @oluoluoxenfree @sanajaved7 @keithamus
```

```suggestion
* @jerivas @oluoluoxenfree @sanajaved7 @keithamus @jgerigmeyer 
```
Adding Jonny from Oddbird to this list as well